src/lib/DateInput.svelte

```diff
@@ -11,32 +11,38 @@
 
   const dispatch = createEventDispatcher<{ select: undefined }>()
 
-  const defaultDate = new Date()
+  /** Default date to use before value is assigned */
+  export let defaultDate = new Date();
 
   // inner date value store for preventing value updates (and also
   // text updates as a result) when date is unchanged
-  const innerStore = writable(defaultDate)
+  const innerStore = writable(null);
```

Comment:
```suggestion
  const innerStore: Date | null = writable(null);
```
We'll need to specify that the value is `Date | null` wherever `Date` is used so there are no TypeScript errors